compiler/modules/front.syntax/token.bal

```diff
@@ -344,12 +347,16 @@ class Tokenizer {
         return createPosition(self.lineIndex, self.tokenStartCodePointIndex);
     }
 
+    function previousStartPos() returns Position {
```

Comment:
I would really prefer to avoid this: prevEndPos was a necessary evil, but it's not a good thing.

Reply:
fixed